app/actions/configurations.py

```diff
@@ -57,7 +57,7 @@ def validate_json(cls, v):
     @pydantic.validator("bounding_box", always=True)
     def validate_bounding_box(cls, v):
         if(not v):
-            raise ValueError("Did not receive a bounding box configuration.")
+            return None
```

Comment:
I see the `bounding_box` field [is defined as a string](https://github.com/PADAS/gundi-integration-inaturalist/blob/967a05886cecff646f60ee3c3125bacaee1e1a68/app/actions/configurations.py#L22), but also later [treated as a list of four](https://github.com/PADAS/gundi-integration-inaturalist/blob/967a05886cecff646f60ee3c3125bacaee1e1a68/app/actions/handlers.py#L50).  So I guess a better default will probably be a list, but a better place to do that may be [in the configuration model](https://github.com/PADAS/gundi-integration-inaturalist/blob/ab9d5a5b8b8fe3b95f386216048c5a0bd300bbe4/app/actions/configurations.py#L11). 

Also, this integration doesn't have unit tests, so I think we will need to add some, or run it and test it locally at least.

Reply:
I also see the code may end up calling float(None) [here](https://github.com/PADAS/gundi-integration-inaturalist/blob/967a05886cecff646f60ee3c3125bacaee1e1a68/app/actions/handlers.py#L63), when the bounding box isn't provided, and that'll raise an exception, so so we'd need to fix that too.